e2e/wdio/platform/pages/checkbox.po.ts

```diff
@@ -27,6 +28,7 @@ export class CheckboxPO extends BaseComponentPo {
     rtlSwitcherArr = 'rtl-switch .fd-switch__handle';
```

Comment:
can be removed, it is now in base-component

Reply:
@Seamoo13 Can you remove rtlSwitcherArr = 'rtl-switch .fd-switch__handle'; ?

Reply:
done